### ISSUE TITLE
[BUG-REPORT] writer_threads is ignored when exporting to hdf5

### ISSUE THREAD

**Issue by dmvieira** [vaexio/vaex#1495]
Thank you for reaching out and helping us improve Vaex!

Before you submit a new Issue, please read through the [documentation](https://docs.vaex.io/en/latest/). Also, make sure you search through the Open and Closed Issues - your problem may already be discussed or addressed.

**Description**
Please provide a clear and concise description of the problem. This should contain all the steps needed to reproduce the problem. A minimal code example that exposes the problem is very appreciated.

**Software information**
 - Vaex version (`import vaex; vaex.__version__)`: 4.3.0
 - Vaex was installed via: pip 
 - OS: ubuntu 20.04

**Additional information**
We're offering this option in API, but writer_threads is not in use

**Comment by maartenbreddels** [comment]
Indeed, we need to pass that argument to `writer.write(..)`, do you want to open a PR for this?
